lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1056,6 +1068,22 @@ LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
       sitofp.getOut().getType().getIntOrFloatBitWidth(), "signedIn");
 }
 
+LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
+                                     DivFOp divf) const {
+  Location loc = divf.getLoc();
+  IntegerType one = rewriter.getI1Type(), three = rewriter.getIntegerType(3),
+              five = rewriter.getIntegerType(5),
+              width = rewriter.getIntegerType(
+                  divf.getType().getIntOrFloatBitWidth());
+  auto divFOp =
+      getState<ComponentLoweringState>()
+          .getNewLibraryOpInstance<calyx::DivSqrtOpIEEE754>(
+              rewriter, loc,
+              {one, one, one, one, one, width, width, three, width, five, one});
```

Comment:
Nit: Instead of five and three, can we say e.g., roundingModeWidth and exceptionalFlagsWidth?

Reply:
sure that'll be more helpful!